preprocessing/preprocess_data.sh

```diff
@@ -112,60 +112,86 @@ cd ${SUBJECT}/anat
 # We do a substitution '/' --> '_' in case there is a subfolder 'ses-0X/'
 file="${SUBJECT//[\/]/_}"
 
-# Add suffix corresponding to contrast
-file=${file}_acq-sag_T2w
+# Add suffix corresponding to contrast (Sagittal T2w)
+file_sag=${file}_acq-sag_T2w
 
-# Make sure the image metadata is a valid JSON object
-if [[ ! -s ${file}.json ]]; then
-  echo "{}" >> ${file}.json
+# Add suffix corresponding to contrast (Axial T2w)
+file_ax=${file}_acq-ax_T2w
+
+# Make sure the Sagittal image metadata is a valid JSON object
+if [[ ! -s ${file_sag}.json ]]; then
+  echo "{}" >> ${file_sag}.json
+fi
+
+# Make sure the Axial image metadata is a valid JSON object
+if [[ ! -s ${file_ax}.json ]]; then
+  echo "{}" >> ${file_ax}.json
 fi
 
-# Spinal cord segmentation using the T2w contrast
-segment_if_does_not_exist ${file} t2 ${CENTERLINE_METHOD}
-file_seg="${FILESEG}"
+# Resample both images to isotropic 1mm x 1mm x 1mm resolution
```

Comment:
why resampling both? you only need to resample one (the 'fixed' image), and then register the source (not resampled) to the dest-- this registration will take care of the resampling to 1mm iso

every time you resample, you introduce interpolation error